src/main/java/telraam/api/LapCountResource.java

```diff
@@ -53,4 +59,16 @@ public Map<String, Integer> getLapCounts() {
 
         return perName;
     }
+
+    // EndTimestamp should be a ISO formatted date timestamp
+    @GET
+    @Path("/{lapSourceId}/{teamId}")
+    public Integer getLapCountForLapSource(@PathParam("lapSourceId") Integer id, @PathParam("teamId") Integer teamId, @QueryParam("end") Optional<String> endTimestamp) {
+        LocalDateTime dateTime = LocalDateTime.now();
+        if (endTimestamp.isPresent()) {
+            dateTime = LocalDateTime.parse(endTimestamp.get());
+        }
+        List<Lap> laps = lapDAO.getAllForTeamBeforeTime(id, teamId, Timestamp.valueOf(dateTime));
+        return laps.size();
+    }
```

Comment:
This is a nice endpoint but only for one team, If we want to move loxsi to using an endpoint like this it should return the info for all teams. I think we do not want to make ~20 request (one for each team) eacht x seconds when refreshing the team score?

Or has this endpoint another main purpose?

Reply:
Well, I was not sure if it should have returned for all team or just one, will refactor

Reply:
Since the code is there, maybe move teamID to a QueryParam?